workflow-templates/command-compile.yml

```
@@ -85,23 +85,28 @@ jobs:
         run: |
           npm ci
           npm run build --if-present
+      
+      - name: Rebase
+        if: ${{ contains(needs.init.outputs.arg1, 'rebase') }}
+        run: |
+          git rebase master
```

Could be a different branch name, you can get the base branch 
```suggestion
          git rebase ${{ github.base_ref }}
```